src/pages/project/helpers/projectContributorHelpers.ts

```diff
@@ -108,3 +114,38 @@ export const addContributor = (
 
   return { newContributors };
 };
+
+export const removeProjectParticipant = (contributors: ProjectContributor[], contributorIndex: number) => {
+  if (contributorIndex < 0) {
```

Comment:
om man skrur paranoiaen til maks kunne det vært en sjekk på om om index er innenfo `contributors.length` også. Men vi har ikke noen særlig praksis med å gjøre det fra før, så trenger ikke for min del